content/guides/developer-guides/delfin/delfin-kubernetes-guide/_index.md

```diff
@@ -91,18 +91,41 @@ Server Version: v1.21.1 <br />
 
 ---
 
-### Build the delfin docker image locally
+### Regarding the delfin image of deployment
```

Comment:
I think u can change the sequence like below
1) Add the section : Get all kubernetes object files required for delfin deployment:
2) After the initial clone part , introduce this "delfin image of deployment"  with all options (default, change version tag, build from source)
3) Then continue from "Commands to bring up the delfin services"

Reply:
changed the order